sympy/parsing/latex/lark/grammar/latex.lark

```diff
@@ -199,7 +199,9 @@ group_curly_parentheses: L_BRACE _expression R_BRACE
 _relation: eq | ne | lt | lte | gt | gte
 
 eq: _expression EQUAL _expression
+    | _relation EQUAL _expression
```

Comment:
It seems to me that this will allow for things like `x > y = z`. I don't think we want to allow such expressions.